config.json

```diff
@@ -0,0 +1,79 @@
+// config.json
+{
+    "influx_db": {
+        "influx_host": "http://localhost:8086",
+        "user": "root",
+        "password": "root",
+        "db": "slow-cooker"
+    },
+    "job": [{
```

Comment:
How is this job being used? I don't see your changes about this. If it's not used yet please take it out from this PR and just keep influx db config.